config/reasoner/dl2op/main/mapping.n3

```diff
@@ -67,25 +67,11 @@ PREFIX lblodlg: <http://data.lblod.info/vocabularies/leidinggevenden/>
 }.
 
 # Test
-# ex:s <http://data.vlaanderen.be/ns/mandaat#start> "2019-01-01T19:32:52+00:00"^^xsd:dateTime.
-{
-  ?s mandaat:start ?startDateTime.
-  # use backward arrow rule defined below
-  ?startDateTime :dateFromDateTime ?startDate.
-  {
-    ?s mandaat:start ?startDateTime
-  } e:becomes {
-    ?s mandaat:start ?startDate
-  }.
-} => {
-  ?s mandaat:start ?startDate
-}.
-
 # ex:s <http://data.vlaanderen.be/ns/mandaat#einde> "2019-02-01T10:32:52+00:00"^^xsd:dateTime.
 {
   ?s mandaat:einde ?eindeDateTime.
   # use backward arrow rule defined below
-  ?eindeDateTime :dateFromDateTime ?eindeDate.
+  ?eindeDateTime :dateTimeFromDate ?eindeDate.
   {
```

Comment:
Loket already uses `dateTime` for `mandaat:einde` (and `mandaat:start`), so this conversion is not necessary. This remark also applies to the same rule in `main-info/mapping.n3`.